hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlPojoTest.java

```diff
@@ -549,12 +519,12 @@ public int hashCode() {
         }
     }
 
+    @SuppressWarnings("unused")
```

Comment:
suppressions are best on the most granular level, only no-arg constructor is not used

Reply:
I also made suppressions more granular in `kafka/SqlPojoTest`, `map/SqlPojoTest` and `BasicNestedFieldsTest`.